kobo/client/__init__.py

```diff
@@ -331,12 +346,13 @@ def _login_gssapi(self):
         if self._conf.get("CA_CERT"):
             request_args["verify"] = self._conf["CA_CERT"]
 
-        server_name = self.get_server_principal(service=service, realm=realm)
-        server_name = gssapi.Name(server_name, gssapi.NameType.kerberos_principal)
+        auth_args = {"mutual_authentication": requests_gssapi.OPTIONAL}
```

Comment:
Did you check whether this affects auth in the non-oidc case?

Reply:
Yeah, I authenticated with gssapi on preprod pub and it worked without an issue